tests/unit/database_tests.py

```diff
@@ -202,6 +202,11 @@ def test_create_db_delete_db(self):
             # No issue should arise if attempting to create existing database
             db_2 = db.create()
             self.assertEqual(db, db_2)
+            # If we use throw_on_exists=True, it will arise a
+            # CloudantDatabaseException if the database already exists.
+            with self.assertRaises(CloudantDatabaseException) as cm:
+                db.create(throw_on_exists=True)
+            self.assertEqual(cm.exception.status_code, 100)
```

Comment:
So there we go @porthunt, looks like your new test inadvertently caught us a bug.